HLTrigger/Configuration/scripts/utils/hltPathOwners.json

```diff
@@ -7939,6 +7988,36 @@
       "BPH"
     ]
   },
+  "HLT_TestData_v": {
+    "online?": true,
+    "owners": [
+      "TSG"
+    ]
+  },
+  "HLT_TriggersForScoutingPFMonitor_PS1000_v": {
+    "online?": true,
+    "owners": [
+      "HLT_Scouting"
+    ]
+  },
+  "HLT_TriggersForScoutingPFMonitor_PS125_v": {
+    "online?": true,
+    "owners": [
+      "HLT_Scouting"
+    ]
+  },
+  "HLT_TriggersForScoutingPFMonitor_PS250_v": {
+    "online?": true,
+    "owners": [
+      "HLT_Scouting"
+    ]
+  },
+  "HLT_TriggersForScoutingPFMonitor_PS500_v": {
+    "online?": true,
+    "owners": [
+      "HLT_Scouting"
```

Comment:
Noticed that this uses `_`, while in other cases we use `-`. I guess it's better to use the same everywhere.